components/arm.py

```diff
@@ -0,0 +1,29 @@
+import wpilib
+
+
+class Arm:
+    """
+    Bunny grabber arm.
+    """
+
+    encoder = wpilib.AnalogPotentiometer
+    motor = wpilib.Talon
+    stop_value = 0
+    extended = False
+
+    def set_stop_value(self, stop_value):
+        self.stop_value = stop_value
+
+    def set_extended(self, extended):
+        self.extended = extended
+
+    def execute(self):
+        if self.encoder.get() > self.stop_value:  # if the arm is overextended
+            self.motor.set(-0.05)  # move arm back slowly
```

Comment:
I realize that _slowly_ refers to the _magical_ `0.05` value, but perhaps we should pull that out as a constant, and then use that value in multiple locations (including our new tests)?

Reply:
Let's put it in a `tunable` so we can access it over `NetworkTables`.

Reply:
That will be replaced with a pid, but whichever pid coefficients we use should definitely be `tunable`.